src/main/resources/profiles/digital-commonwealth.conf

```diff
@@ -11,6 +11,5 @@ endpoint = "http://fedora.digitalcommonwealth.org/oaiprovider/"
 metadataPrefix = "mods"
 outputDir = "" # TODO This should be an S3:// bucket that i3 has write access to
 provider = "Digital Commonwealth"
-type = "oai" # Which harvester to invoke
```

Comment:
It's clear that we only have one harvester so far, OAI-PMH, but won't there be others at some point, such that the `type` option still has a place?